ecocode-rules-specifications/src/main/rules/EC86/csharp/EC86.asciidoc

```diff
@@ -0,0 +1,44 @@
+:!sectids:
+
+GC.Collect should not be called.
+
+## Why is this an issue ?
+
+Calling GC.Collect forces a garbage collection to occur. This can be a very expensive operation, as it is fully blocking and can take a significant amount of time to complete.
+In most cases, the garbage collector is able to determine when it is appropriate to run a collection, and calling GC.Collect is not only unnecessary but also not advisable.
+
+Generation 0 objects, so called ephemeral objects, are excluded from this analysis because they are collected very frequently and are usually not a concern.
```

Comment:
I'm not sure to understand this sentence.
Are we speaking about objects from generation "0" ?

Reply:
Yes, calling GC.Collect(generation: 0) will not raise a warning because that's mostly harmless.
If the sentence is not clear enough I can rephrase it.

Reply:
Ok I've reworded the sentence, it should be clearer